data/mods/Magiclysm/monstergroups.json

```diff
@@ -16,6 +16,9 @@
       { "monster": "mon_troll", "freq": 3, "cost_multiplier": 10, "conditions": [ "DAY" ] },
       { "monster": "mon_claygolem", "freq": 8, "cost_multiplier": 3 },
       { "monster": "mon_stonegolem", "freq": 4, "cost_multiplier": 5 },
+      { "monster": "mon_yulecat", "freq": 2, "cost_multiplier": 7 },
```

Comment:
I would delay these from appearing until late fall or winter.  Let me know if you need help finding the json to do that.  With as fast and tough as they are can a day one character escape from them?

Reply:
Will adding conditions: winter work for this. ( Like how troll has conditions: day)?